service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AIS.feature

```diff
@@ -16,6 +16,14 @@ Feature: AIS
       | null                                          | DE94500105178833114935 | null                   | 200  |
       | DE94500105178833114935;DE96500105179669622432 | DE94500105178833114935 | null                   | 200  |
 
+  Scenario Outline: PSU Validation AIS
+    Given PSU created a consent on dedicated accounts for account information <accounts>, balances <balances> and transactions <transactions>
+    When PSU tries to authorise the consent with psu-id <psu-id>, password <password>
```

Comment:
It's not clear, that the PSU (line 20 + 21) are two different PSUs. Maybe write "Another PSU .." in line 21